blocks/init/src/Blocks/components/button/manifest.json

```diff
@@ -62,7 +62,7 @@
 		}
 	},
 	"options": {
-		"sizes": [
+		"buttonSize": [
```

Comment:
Does this have to be prefixed with the `button`? Seems kinda redundant as we are in the button manifest 👀

Reply:
Yes, because without that this wont work. The problem is that you can have options name buttonSize or introSize because of the components in manifest. Without this you would need to map additional key in the helper. And with this way you know exactly for that this options is used

Reply:
Kinda ugly, but makes sense xD